search-scores-and-weights.md

```diff
@@ -0,0 +1,40 @@
+Our search system uses scoring to assign values to each document based on
+different parameters. The interplay between clustering and scoring helps
```

Comment:
clustering is an indexing method. eventually, other ways to organize related information could be found. 

semantic similarity is what we want to explain as the first signal that we use (similarity is itself a score, but instead of precomputed it is computed on the fly from the search results).

Reply:
added some stuff on that